apps/client/src/components/Questions.js

```diff
@@ -90,17 +102,43 @@ const Questions = ({
     [goToQuestion, setActiveState, setFinishedState]
   );
 
+  // @TODO: Move to checker.js
+  const getUserAnswer = (question) => {
+    const booleanAnswers =
+      !question.options &&
+      booleanOptions.find((o) => o.value === question.answer);
+    return question.options ? question.answer : booleanAnswers?.label;
+  };
+
+  const shouldGoToConlusion = () => {
+    if (!checker.isConclusive()) {
+      return false;
+    } else if (contactConclusion) {
+      return true;
+    }
+
+    let isCompleted = true;
+
+    // Go through all questions and check if they are answered
+    // There is still in bug _getUpcomingQuestions() where some irrelevant questions are unanswered
+    checker.stack.concat(checker._getUpcomingQuestions()).forEach((q) => {
```

Comment:
I would try a `return !!checker.stack.concat(checker._getUpcomingQuestions()).find` and stop on the first occurrence of isCompleted = false. Return it directly it by double negating it.

Reply:
👍